src/components/CheckingAccountCreation.jsx

```diff
@@ -0,0 +1,118 @@
+                        <input onChange={handleInputChange} name="accountType" className="form-check-input ms-3" type="radio" value="SCK" id="flexCheckDisabled" checked={account.accountType === "SCK"} />
+                        <label className="ms-1"> Standard checking</label>
+
+                    </div>
+
+                    <div className="d-inline">
+                        <input onChange={handleInputChange} name="accountType" className="form-check-input ms-3" type="radio" value="ICK" id="flexCheckDisabled" checked={account.accountType === "ICK"} />
+                        <label className="ms-1"> Interest checking</label>
+
+                    </div>
+
+                    
+
+                </div>
+
+                <div>
+                    <strong className="d-block mt-4 ms-3 mb-2">Select Account Ownership</strong>
+                    <div className="d-inline">
+                        <input onChange={handleInputChange} name="accountOwnership" className="form-check-input ms-3" type="radio" value="IND" id="flexCheckDisabled" checked={account.accountOwnership === "IND"} />
+                        <label className="ms-1"> Individual</label>
+
+                    </div>
+
+                    <div className="d-inline">
+                        <input onChange={handleInputChange} name="accountOwnership" className="form-check-input ms-3" type="radio" value="JNT" id="flexCheckDisabled" checked={account.accountOwnership === "JNT"} />
+                        <label className="ms-1"> Joint</label>
+
+                    </div>
+
+                    
+
+                </div>
+
+                <div className="ms-3 mt-4">
+                    <strong>Account Name</strong>
+                    <input data-testid="account-name" type="text" onChange={handleInputChange} autoComplete="off" name="accountName" className="form-control w-75" value={account.accountName}/>
+
+                </div>
+                <div className="ms-3 mt-4">
+                    <strong>Initial deposit</strong>
+                    <input data-testid="initial-deposit" type="text" onChange={handleInputChange} autoComplete="off" name="accountOpening" className="form-control w-75" value={account.accountOpening} />
+                    <span className={`${depositError ? 'text-danger' : 'text-dark'}`}>Minimum opening deposit is $25.00</span>
+
+                </div>
+
+                <div className="ms-3 mt-4">
+                    <button data-testid="submit" type="submit" className="btn btn-success">Create</button>
+                    <button onClick={resetForm} className="btn btn-danger ms-3">Cancel</button>
+
+                </div>
+                
+                
+
+
+            </form>
+            {
+                    successAlert && <div data-testid="success" style={{marginLeft:400}} className="alert alert-success w-50 mt-3">The account has been created</div>
+            }
+        </div>
+
+    )
+}
```

Comment:
Remember to use Prettier to auto-format your code ;)

Reply:
I´m gonna install it